### ISSUE TITLE
Backup image for twitter avatars?

### ISSUE THREAD

**Issue by snarfblam** [TheSamLloyd/Bootcamp-Project-1#9]
Noticed for some reason the pope's avatar is coming up as a 404. We might want to insert a placeholder in such an event, ala https://stackoverflow.com/questions/34243007/backup-img-if-the-first-cant-be-found-on-html

**Comment by TheSamLloyd** [comment]
yeah you're totally right -- i happen to know offhand WHY the pope's avatar is coming up blank (its a bug in the fix we use to get the higher-res images) but having a backup to get the lower res one and then a backup to use in place of that is a good idea
